jax/lax/lax.py

```diff
@@ -1701,6 +1719,9 @@ def _sign_translation_rule(c, x):
   lambda g, x, y: _brcast(g, y) * (y / (square(x) + square(y))),
   lambda g, x, y: _brcast(g, x) * -x / (square(x) + square(y)))
 
+regularized_incomplete_beta_p = standard_terop(
+    [_float, _float, _float], 'regularized_incomplete_beta')
+
```

Comment:
It looks like it wouldn't be hard to define a JVP rule for betainc too? TF's gradient appears to be in terms of `lgamma`, which is available here.

Reply:
Is there a way to mark it for one variable? The gradient for x is defined, but the gradient for a and b require a custom special function

Reply:
The closest would be to raise an exception if the unhandled tangent inputs are anything other than the object `ad_util.zero`.

Reply:
Added it for x, and raised an exception. Is this what you had in mind?